webapi/src/main/scala/org/knora/webapi/responders/v1/PermissionsResponderV1.scala

```diff
@@ -0,0 +1,853 @@
+
+            permission: Option[DefaultObjectAccessPermissionV1] = if (permissionQueryResponseRows.nonEmpty) {
+
+                /* check if we only got one default object access permission back */
+                val doapCount: Int = permissionQueryResponseRows.groupBy(_.rowMap("s")).size
+                if (doapCount > 1) throw InconsistentTriplestoreDataException(s"Only one default object permission instance allowed for project: $projectIRI and group: $groupIRI combination, but found $doapCount.")
+
+                /* get the iri of the retrieved permission */
+                val permissionIri = permissionQueryResponse.getFirstRow.rowMap("s")
+
+                val groupedPermissionsQueryResponse: Map[String, Seq[String]] = permissionQueryResponseRows.groupBy(_.rowMap("p")).map {
+                    case (predicate, rows) => predicate -> rows.map(_.rowMap("o"))
+                }
+                val hasPermissions: Set[PermissionV1] = PermissionUtilV1.parsePermissions(groupedPermissionsQueryResponse.get(OntologyConstants.KnoraBase.HasPermissions).map(_.head), PermissionType.OAP)
+                Some(
+                    DefaultObjectAccessPermissionV1(iri = permissionIri, forProject = groupedPermissionsQueryResponse.getOrElse(OntologyConstants.KnoraBase.ForProject, throw InconsistentTriplestoreDataException(s"Permission has no project.")).head, forGroup = groupedPermissionsQueryResponse.get(OntologyConstants.KnoraBase.ForGroup).map(_.head), forResourceClass = groupedPermissionsQueryResponse.get(OntologyConstants.KnoraBase.ForResourceClass).map(_.head), forProperty = groupedPermissionsQueryResponse.get(OntologyConstants.KnoraBase.ForProperty).map(_.head), hasPermissions = hasPermissions)
+                )
+            } else {
+                None
+            }
+            //_ = log.debug(s"defaultObjectAccessPermissionGetV1 - p: $projectIRI, g: $groupIRI, r: $resourceClassIRI, p: $propertyIRI, permission: $permission")
+        } yield permission
+    }
+
+    /**
+      * Gets a single default object access permission identified by project and either group / resource class / property.
+      * In the case of properties, an additional check is performed against the 'SystemProject', as some 'knora-base'
+      * properties can carry default object access permissions. Note that default access permissions defined for a system
+      * property inside the 'SystemProject' can be overridden by defining them for its own project.
+      *
+      * @param projectIRI
+      * @param groupIRI
+      * @param resourceClassIri
+      * @param propertyIri
+      * @param userProfile
+      * @return a [[DefaultObjectAccessPermissionGetResponseV1]]
+      */
+    def defaultObjectAccessPermissionGetRequestV1(projectIRI: IRI, groupIRI: Option[IRI], resourceClassIri: Option[IRI], propertyIri: Option[IRI], userProfile: UserProfileV1): Future[DefaultObjectAccessPermissionGetResponseV1] = {
+
+        // FIXME: Check user's permission for operation
```

Comment:
Is this already fixed? If so, please delete the comment. Otherwise, make an issue for it on GitHub.